src/sentry/static/sentry/app/utils/theme.jsx

```diff
@@ -87,6 +87,9 @@ const theme = {
   },
 
   grid: 8,
+  fontSizeSmall: '12px',
+  fontSizeMedium: '16px',
+  fontSizeLarge: '18px',
```

Comment:
these are maybe not ideal but it's better than nothing, and as of today calling stuff nested inside the text object will not pass tests.

Reply:
Agreed. 👍on having them here for now.